jwt/keyserver/keyregistry/keyregistry.go

```diff
@@ -57,16 +71,15 @@ func (krc *client) GetPublicKey(issuer string, keyID string) (*key.PublicKey, er
 	if err != nil {
 		return nil, err
 	}
-
-	if resp.StatusCode == http.StatusNotFound {
+	switch resp.StatusCode {
```

Comment:
This stuff belongs in the autogenerated module. There are other uses of the key registry GetPublicKey that are unrelated.